src/Entity/Log/HistorizableTrait.php

```diff
@@ -35,4 +35,22 @@ public function addHistory(array $history): self
 
         return $this;
     }
+
+    public function getDataForHistoryDiff(): array
+    {
+        $toReturn = [];
+        foreach ($this->getAttributsForHistoryDiff() as $attribut) {
+            $toReturn[$attribut] = $this->{'get' . ucwords($attribut)}();
```

Comment:
@lfortunier does this still work with camelCase Attribut ?
Like myVariable will translate to getMyvariable and i don't know if this will work suppose that the entity will have a getMyVariable method

Reply:
juste to be sure, use the PropertyAccess technique to get the value